packages/server/src/service/alloy.proto.ts

```diff
@@ -0,0 +1,35 @@
+import { alloy } from './alloy'
+import { StateInfo, processEnvOrThrow } from '../common'
+
+const dummy: StateInfo = {
```

Comment:
Can we have details for one voter who is registered and one who is not (`registration_status`)?

Also, can you document that these are from here: https://docs.alloy.us/api/#section/Sandbox/Getting-Started